Application/EdFi.Ods.Features/Publishing/Resources/Snapshot.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Serialization;
+using EdFi.Ods.Common;
+using Newtonsoft.Json;
+
+namespace EdFi.Ods.Features.Publishing.Resources
+{
+    /// <summary>
+    /// A class which represents the publishing.Snapshot table of the Snapshot aggregate in the ODS Database.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class Snapshot
+    {
+        /// <summary>
+        /// The unique identifier for the Snapshot resource.
+        /// </summary>
+        [DataMember(Name="id")]
```

Comment:
We ran into this with change queries, these DataMember attributes won't do anything without DataContract on the class.  They're not needed anyways, since the default json serializer settings will already name them this way, lower casing the first letter.

Reply:
Done.